dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -638,6 +638,7 @@ private List<HierarchedCategory> convertForHierarchedCategories(final List<Map<S
 						final String parentsASJsonArray = "[" + row.get("path") + "]";
 						final List<ShortCategory> parentList = getShortCategories(parentsASJsonArray);
 						category.setParentList(parentList.subList(0, parentList.size() - 1));
+						category.setChildrenCount(Integer.parseInt(row.get("childrencount").toString()));

```

Comment:
we have a method on ConversionUtils for it safe

Reply:
done https://github.com/dotCMS/core/pull/29251/commits/a68facb0ae34777533e38d095d0eb31f0bed1f17#diff-bf828c747c99cefe73af7cec527db7f7e64bd4a66ba54a55eb1170f6a2996333R638